exercises-python/Gui_Pereira/ex01_basics/main.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import sys
+
+
+def main():
+    # file_path = Path(__file__).parent.absolute() / 'holmes.txt'
+    file_path = Path(sys.argv[1])
```

Comment:
Could you add an error message to handle an argument not provided, or the file not able to be found?